### PR TITLE
black: magic trailing comma

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

please vote for using pure black :+1: or with skip comma :-1:
note that the main argument for black was standard code not customize it as yapf and if someone chose, it is on purpose to make code readable (compare to black)

see: https://github.com/PyTorchLightning/pytorch-lightning/pull/7783#discussion_r643109384

## Before submitting

- [ ] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [x] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -21,7 +21,7 @@ order_by_type = "False"
 
 [tool.black]
 line-length = 120
-skip-magic-trailing-comma = true
+skip-magic-trailing-comma = false
```
Comment (awaelchli) [inline pyproject.toml:24]: this is the default, right? so we can just delete it.

**Review by kaushikb11 — APPROVED** [review]

**Review by awaelchli — APPROVED** [review]

**Review by carmocca — DISMISSED** [review]
This was already discussed in https://github.com/PyTorchLightning/pytorch-lightning/pull/7783#discussion_r643088871

If you want to change it, lets have a poll with the wider team. I would like to avoid going back-and-forth between options

**Comment by Borda** [comment]
> This was already discussed in [#7783 (comment)](https://github.com/PyTorchLightning/pytorch-lightning/pull/7783#discussion_r643088871)
> 
> If you want to change it, lets have a poll with the wider team. I would like to avoid going back-and-forth between options

yep, I see that it was 2 in favour and 2 against...
but if I see it correctly, the `skip-magic-trailing-comm` is not default

**Review by tchaton — APPROVED** [review]
LGTM !

**Pull request merged**